EasyPost.Tests/Sample.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
```

Comment:
Question: What is this file's purpose?

Reply:
Just a scratch file to test and show off the redesign.

Reply:
🤷‍♂️ let's remove this. People can copy/paste the README example to test now that we know it works.

Reply:
Resolved.